src/string_object.cpp

```diff
@@ -2256,6 +2256,38 @@ Value StringObject::upcase(Env *env) {
     return str;
 }
 
+Value StringObject::capitalize(Env *env) {
+    auto str = new StringObject { "", m_encoding };
+    if (length() == 0) {
+        return str;
+    } else {
+        auto ary = chars(env)->as_array();
+        auto first_char = ary->shift()->as_string();
+        if (first_char->bytesize() > 1) {
+            str->append(first_char);
+        } else {
+            auto c = first_char->c_str()[0];
+            if (c >= 97 && c <= 122) {
```

Comment:
You can use 'a' and 'z' as char values as well, which makes it a bit more readable:
```c
if (c >= 'a' && c <= 'z') {
```